client/package.json

```diff
@@ -6,7 +6,7 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint",
+    "lint": "next lint --report-unused-disable-directives warn",
```

Comment:
Just have a question. Why do we need this flag?

Reply:
We don't need it per se, but I think it's nice to warn when some directives disable warnings/errors that don't exist anymore. The same way we warn with unused variables and such